api/src/models/IUser.ts

```diff
@@ -0,0 +1,7 @@
+export interface IUser {
+    id:string,
+    name:string,
+    email:string,
+    salt:string,
+    password:string
```

Comment:
Again, seems to me that we shouldn't even store the password hash (or active JWT token) in any model because it just risks leaking it through an API endpoint

Reply:
It should be removed now